tests/test_auth.py

```diff
@@ -10,114 +10,83 @@
 
 import python_pachyderm
 from tests import util
-
+from python_pachyderm.proto.auth import auth_pb2
+from python_pachyderm.proto.identity import identity_pb2
 
 @contextmanager
```

Comment:
Not a blocker, but instead of `@contextmanager`, we could try using `@pytest.fixture`, which provides some nice features like safe teardowns.